src/views/List.jsx

```diff
@@ -1,15 +1,45 @@
 import { ListItem } from '../components';
+import { useState } from 'react';
 
 export function List({ data }) {
+	const [searchItem, setSearchItem] = useState('');
+
+	const handleTextChange = (event) => {
+		setSearchItem(event.target.value);
+	};
+
+	const filteredItems = data.filter((item) => {
+		return item.name.toLowerCase().includes(searchItem.toLowerCase());
+	});
```

Comment:
Non-blocking: totally up to you if you want to keep this as is, but you could shorten this a bit too if you want.


```suggestion
	const filteredItems = data.filter(item => item.name.toLowerCase().includes(searchItem.toLowerCase()));
```